datajunction-server/datajunction_server/api/graphql/scalars/node.py

```diff
@@ -182,3 +183,10 @@ class Node:  # pylint: disable=too-few-public-methods
 
     tags: List[Tag]
     created_by: User
+
+    @strawberry.field
+    def edited_by(self, root: "DBNode") -> List[str]:
+        """
+        The users who edited this node
+        """
+        return list({entry.user for entry in root.history})
```

Comment:
Should we distinctify this list before returning?

Reply:
Oh it's actually already distinct because of the set comprehensive `{ ... }`